### PR TITLE
DOCSP-30082: v1.9 doc to ref

### PR DESCRIPTION
**Pull request by nickldp**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-30082>
Staging - <https://docs-mongodbcom-staging.corp.mongodb.com/golang/docsworker-xlarge/DOCSP-30082-v1.9-doc-to-ref/>

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: source/fundamentals/crud/read-operations/query-document.txt [hunk 1]
```diff
@@ -443,8 +441,8 @@ Query Operators </reference/operator/query-bitwise/>` page.
 Additional Information
 ----------------------
 
-.. For information on specifying a geospatial query, see the guide on
-.. :doc:`Geospatial Data </fundamentals/crud/read-operations/geospatial>`.
+For information on specifying a geospatial query, see the guide on
+:ref:`golang-geo`.
```
Comment (rustagir) [inline source/fundamentals/crud/read-operations/query-document.txt:445]: Suggestion: Uncomment and add link text:
```suggestion
For information on specifying a geospatial query, see the guide on
:ref:`Geospatial Data <golang-geo>`.
```

**Review by rustagir — CHANGES_REQUESTED** [review]
one small fix!

File: source/fundamentals/crud/read-operations/query-document.txt (hunk 1 above)
Comment (nickldp) [reply source/fundamentals/crud/read-operations/query-document.txt]: back to you!

**Review by rustagir — APPROVED** [review]
lgtm!

**Pull request merged**